client/src/main.tsx

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import App from "./App";
 import "./index.css";
+import { router } from "./router";
+import {RouterProvider} from "react-router-dom"
```

Comment:
lets structurize our way of imports 

we will follow the following way from now onwards :-

```
3rd party imports  ('react' / 'rect-dom', 'ant-design', etc)
<\n>
alias / absolute imports (@someFolder/file.js)
<\n>
relative imports (./files.js)

```